services/director-v2/tests/unit/test_models_clusters.py

```diff
@@ -52,3 +53,24 @@ def test_scheduler_constructor_with_no_workers_has_correct_dict(faker: Faker):
     scheduler = Scheduler(status=faker.text(), workers=None)
     assert isinstance(scheduler.workers, WorkersDict)
     assert len(scheduler.workers) == 0
+
+
+def test_worker_constructor_corrects_negative_used_resources(faker: Faker):
+    worker = Worker(
+        id=faker.pyint(min_value=1),
+        name=faker.name(),
+        resources={},
+        used_resources={"CPU": -0.0000234},
+        memory_limit=faker.pyint(min_value=1),
+        metrics={
+            "cpu": faker.pyfloat(min_value=0),
+            "memory": faker.pyint(min_value=0),
+            "num_fds": faker.pyint(),
```

Comment:
num_fds can be negative? 
ready, executing, etc ... are not all also non-negative?

Reply:
from definition of file descriptor in wikipedia:
```File descriptors typically have non-negative integer values, with negative values being reserved to indicate "no value" or error conditions.```

Reply:
for the tasks I will put them as nonnegative since it seems that dask is not doing operations on them